resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/rest/kubernetes/HttpClientUtil.scala

```diff
@@ -33,6 +33,7 @@ private[spark] object HttpClientUtil {
 
   def createClient[T: ClassTag](
       uris: Array[String],
+      maxRetriesPerServer: Int = 1,
```

Comment:
does this param need to get passed through to the MultiServerFeignTarget constructor?